HIRS_Utils/src/main/java/hirs/data/persist/certificate/PlatformCredential.java

```diff
@@ -73,6 +72,13 @@
     private static final String TCG_CREDENTIAL_SPECIFICATION = "2.23.133.2.23";
     private static final String PLATFORM_CONFIGURATION_URI = "2.23.133.5.1.3";
     private static final String PLATFORM_CONFIGURATION = "2.23.133.5.1.7.1";
+    private static final String PLATFORM_CONFIGURATION_V2 = "2.23.133.5.1.7.2";
+
+    //TCG Platform Specification values
+    private static final Map<String, String> TCG_PLATFORM_MAP = new HashMap<String, String>() {{
+        put("#00000000", "Client");
+        put("#00000001", "Server");
```

Comment:
Along with the above comment, are we positive these values are accurate? The `PlatformCredentialTest` that was modified had Credentials giving a value of `1` for `platformClass`, which I would think corresponds to `Server` here. Now the test claims that they have a null `platformClass`, which isn't exactly true.

Reply:
Nope we are not, and actually they aren't because everything comes up at server when they are clearly client certificates.  These code's main function right now is to just have something displayed on the portal.  The platformClass variable was code that was present but not displayed, and what was displayed was the numerical value.